### PR TITLE
SONAR-6887 Make it clear in SCM API that date and revision are mandat…

### PR DESCRIPTION
**Pull request by dbmeneses**
…ory on line blame

### CODE REVIEW

File: sonar-plugin-api/src/main/java/org/sonar/api/batch/scm/BlameLine.java [hunk 1]
```diff
@@ -37,11 +37,22 @@
   private Date date;
   private String revision;
   private String author;
+  
+  public BlameLine() {
```
Comment (SonarTech) [inline sonar-plugin-api/src/main/java/org/sonar/api/batch/scm/BlameLine.java]: MAJOR Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [rule](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1186)

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 issue:
* MAJOR 1 major

Watch the comments in this conversation to review them.

**Comment by henryju** [comment]
LGTM

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 issue:
* MAJOR 1 major

Watch the comments in this conversation to review them.

**Pull request closed without merging**